packages/rum/package.json

```diff
@@ -3,23 +3,14 @@
   "version": "3.0.0",
   "description": "Elastic APM JavaScript agent",
   "main": "dist/bundles/elastic-apm-js-base.umd.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/elastic/apm-agent-js-base.git"
-  },
+  "repository": "https://github.com/elastic/apm-agent-js-base/tree/master/packages/rum",
   "scripts": {
-    "prepare-patch": "sh ../../prepare-release patch && npm run build && npm run release",
```

Comment:
all of the release scripts are managed via lerna

+ creating tags
+ .writing changelog
+ updating package.json
+ publishing to npm